group-membership/src/Code.js

```diff
@@ -0,0 +1,111 @@
+var EMAIL = 'Email';
+var GOOGLE_GROUP = 'Google Group';
+var ALLOWED = 'Allowed';
+var EMAIL_TEMPLATE_DOC_URL = 'Email template doc URL';
+var EMAIL_SUBJECT = 'Email subject';
+var STATUS = 'Status';
+
+/**
+ * Trigger that runs on edit after being installed via the interface.
+ *
+ * @param {Object} e - onEdit trigger event.
+ */
+function onEditInstallableTrigger(e) {
```

Comment:
Either need some form of validation to make sure the user input the full set of required fields with reasonable input OR stick a form in front of the Sheet

Reply:
Roger that, added validation in the status column to say that all fields in the row need to be populated.

Reply:
Apologies, this applies more to addToGroup